README.md

```diff
@@ -17,240 +20,154 @@ of the [Privacy Community Group](https://privacycg.github.io/).
-* Show browser UI acquiring user intent when login status is set.
-  Example: A prompt.
-* Continuously show browser UI indicating an active login session on
-  the particular website. Example: Some kind of indicator in the URL
-  bar.
-* Delayed browser UI acquiring user intent to stay logged in, shown some
-  time after the login status was set. Example: Seven days after login
-  status was set – “Do you want to stay logged in to news.example?”
-* Requiring engagement to maintain login status. Example: Require user
-  interaction as first party website at least every N days to stay 
-  logged in. The browser can hide instead of delete the credential token
-  past this kind of expiry to allow for quick resurrection of the login
-  session.
-
-### Credential Tokens
-
-Ideally, a new Logon Status API like this would only work with modern
-login credentials. The proposed 
-[HTTP State
-Tokens](https://mikewest.github.io/http-state-tokens/draft-west-http-state-tokens.html)
-could become such a modern piece. However, to ensure a smooth path for
-adoption, we should support cookies.
-
-Both HTTP State Tokens and cookies would have to be explicitly set up
-for authentication purposes to work with the Login Status API. In the
-case of both of these token types, we could introduce an __auth- prefix
-as a signal that both the server and client consider the user to be
-logged in. Or we could allow HTTP State Token request and response
-headers to convey login status. Note that sending metadata in requests
-differs from how cookies work.
-
-The expiry of the cookie or token should be picked up as recordLogout().
-
-Cookies have the capability to span a full registrable domain and thus
-log the user in to all subdomains at once. HTTP State Tokens have a
-proper connection to origins but can be declared to span the full
-registrable domain too. We should probably let the credential token
-control the scope of the login status.
-
-Explicitly logging out should clear all website data for the website,
-not just the credential token. The reverse, the user clearing the
-credential token (individually or as part of a larger clearing of
-website data), should also log them out for the purposes of the Login
-Status API.
-
-### Federated Logins
-
-Some websites allow the user to use an existing account with a federated
-login provider to bootstrap a new local user account and subsequently
-log in. The Login Status API needs to support such logins.
-
-This could be achieved through integration between the Login Status API
-and the proposed [WebID](https://github.com/WICG/WebID).
-
-If federated logins were to be supported by the Login Status API alone,
-it could look like this:
-
-First, the federated login provider needs to call the API on its side,
-possibly after the user has clicked a “Log in with X” button:
+#### HTTP API
+
+Here is how a HTTP server could operate on the signals:
 
+```javascript
+Sign-in-Status: type=idp, action=signin
```

Comment:
fwiw, this needs to be ; instead of ,

(https://www.rfc-editor.org/rfc/rfc9110#parameter)

Reply:
I'd also prefer Signin-Status over Sign-in-status